app/controllers/films_controller.rb

```diff
@@ -11,7 +28,7 @@ def index
       'X-RapidAPI-Key' => '562c0c79a0msha9b8db4de63e59dp131a2djsna425bbbdbb01'
     })
     @movie_detail = JSON.parse(response.body)['contents']
-    @new_movies = @movie_detail.reject { |movie| @films.exists?(title: movie['title']) }
+    @new_movies = @movie_detail.reject { |movie| @films.exists?(title: movie['title']) } if @movie_detail.present?
```

Comment:
you can fail the save operator instead of if block here use `&`
`@movie_detail&.reject`